sig-usability/README.md

```diff
@@ -13,8 +13,8 @@ The scope of SIG usability is the core end-user usability of the Kubernetes proj
 The [charter](charter.md) defines the scope and governance of the Usability Special Interest Group.
 
 ## Meetings
-* Regular SIG Meeting: [TBDs at TBD PT (Pacific Time)](tbd) (every third week). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=TBD&tz=PT%20%28Pacific%20Time%29).
-  * [Meeting notes and Agenda](https://docs.google.com/document/d/1-dYznZR7Q5XCd2bVYoNfMPYs4uNn1aaePQF0q0JQ0uU/).
+* Regular SIG Meeting: [2 PM (Pacific Time)](https://zoom.us/j/3832562240 ) (every third week). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=TBD&tz=PT%20%28Pacific%20Time%29).
```

Comment:
This needs to be updated in sigs.yaml first and then this file is updated automatically after running `make generate` :)

https://github.com/kubernetes/community/blob/2d82d25e3e6c135ac2b57892fbc2dff005074bc5/sigs.yaml#L2093-L2101

Reply:
I realized after I pushed the changes. Thanks for reminding :)